recipes/natural_language_processing/chatbot-nodejs/ai-lab.yaml

```
@@ -0,0 +1,27 @@
+version: v1.0
+application:
+  type: language
+  name: ChatBot_nodejs
+  description: Chat with a model service in a web frontend.
+  containers:
+    - name: llamacpp-server
+      contextdir: ../../../model_servers/llamacpp_python
+      containerfile: ./base/Containerfile
+      model-service: true
+      backend:
+        - llama
```

Shouldnt this be:
```suggestion
        - llama-cpp
```
as per: https://github.com/containers/ai-lab-recipes/blob/main/recipes/natural_language_processing/chatbot/ai-lab.yaml#L12

---

Looks like it was changed since I put the PR together, accepted the suggestion.